BackofficeBundle/Resources/public/coffee/page/node/NodeRouter.coffee

```diff
@@ -0,0 +1,48 @@
+###*
+ * Router node
+###
+((router) ->
+
+  ###*
+   * show node
+  ###
+  router.route 'node/show/:nodeId', 'showNode', (nodeId) ->
+    @showNodeAction(nodeId)
+
+  ###*
+   * show node with language
+  ###
+  router.route 'node/show/:nodeId/:language', 'showNodeWithLanguage', (nodeId, language) ->
+    @showNodeAction(nodeId, language)
+    return
+
+  ###*
+   * show node with language and version
+  ###
+  router.route 'node/show/:nodeId/:language/:version', 'showNodeWithLanguageAndVersion', (nodeId, language, version) ->
+    @showNodeAction(nodeId, language, version)
+    return
+
+  ###*
+   * load node and display node view
+  ###
+  router.showNodeAction = (nodeId, language = null, version = null)->
+    @initDisplayRouteChanges '#nav-node-' + nodeId
+    url = $('#nav-node-' + nodeId).data('url')
+    url = url + '?language=' + language if language?
```

Comment:
@alavieille : this is not good, but I wonder how to make it clean. We already have the case elsewhere, and Symfony generates a standard route with fake paramters and inject it into the div hidden info. Then the js parse it and replace parameters dynamically. That ways the js doesn't have to know the server routes, but it's uggly too :/